test/cpp/jit/test_custom_class.cpp

```diff
@@ -83,8 +75,14 @@ static auto testStack =
         .def("pop", &Stack<std::string>::pop)
         .def("clone", &Stack<std::string>::clone)
         .def("merge", &Stack<std::string>::merge)
-        .def("__getstate__", &Stack<std::string>::__getstate__)
-        .def("__setstate__", &Stack<std::string>::__setstate__)
+        .def(torch::jit::pickle_(
```

Comment:
people will confuse torch::jit::pickle with torch::jit::pickle_. Compromise would be to make it `.def_pickle(get, set);` which is close enough to pybind but clearly different from the pickler.